src/transformers/integrations.py

```diff
@@ -98,6 +98,10 @@ def is_tensorboard_available():
     return importlib.util.find_spec("tensorboard") is not None or importlib.util.find_spec("tensorboardX") is not None
 
 
+def is_optimum_available():
+    return importlib.util.find_spec("optimum") is not None
```

Comment:
No need to redefine it it already exists in `import_utils`.

Reply:
Thanks, fixed in [9c5e0d9](https://github.com/huggingface/transformers/pull/21259/commits/9c5e0d9308b5c4882805058023729fb6dd63cbe9)